msal/src/Microsoft.Identity.Client/Internal/PlatformPlugin.cs

```diff
@@ -45,21 +45,21 @@ static PlatformPlugin()
 
         public static void InitializeByAssemblyDynamicLinking()
         {
+            CoreExceptionFactory.Instance = new MsalExceptionFactory();
```

Comment:
>CoreExceptionFactory.Instance = new MsalExceptionFactory(); [](start = 12, length = 59)

minor: as discussed perhaps moving to an actual assembly loaded event as that reveals the intent much better than this... it's a bit of magic that this happens before anything else is called

Reply:
Will address this and an AdalFactory in a separate PR.